Java/src/main/java/Planes/MilitaryPlane.java

```diff
@@ -6,35 +6,34 @@
 
 public class MilitaryPlane extends Plane{
 
-    private MilitaryType type;
+    private MilitaryType militaryType;
 
-    public MilitaryPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity, MilitaryType type) {
+    public MilitaryPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity, MilitaryType militaryType) {
```

Comment:
In real world you should replace constructors like this with Builders. Here is what Effective Java has to say about this
https://www.informit.com/articles/article.aspx?p=1216151&seqNum=2